Generator/ImageFileHandler.php

```diff
@@ -74,7 +62,7 @@ public function saveAsFile($contents): string
 
     public function collectGarbage(): bool
     {
-        if (1 == !mt_rand(1, $this->gcFreq)) {
+        if (1 != mt_rand(1, $this->gcFreq)) {
```

Comment:
@Gregwar I think this had to be a long-time bug in the code, as the old code would be ALWAYS false. Hence the garbage collection was running EVERY SINGLE TIME. But check, maybe I didnt understant properly

Reply:
I think you should use strict comparison here
```suggestion
        if (1 !== mt_rand(1, $this->gcFreq)) {
```